### ISSUE TITLE
Feedback XMage Version 1.4.31V1

### ISSUE THREAD

**Issue by LevelX2** [magefree/mage#5335]
# 1.4.30V1

- [ ] Yuriko, the Tiger's Shadow: Commander ninjitsu still does not work. Just cancels cast after paying man. Resets mana as well (reported by Iwanabtheguy / 2018-08-26 11:29:19.074).
- [ ] Yuriko still doesn't work: Yuriko needs niinjutsu to work from the command zone (reported by RustyShack / 2018-08-26 15:38:20.784).
- [ ] please ban catdbo for using hate speech excesssively: (300 characters max)calling everyone faggots (reported by bentspace / 2018-08-26 16:55:58.663).
- [ ] steel hellkite fix: think steel hellkite is removing permanents from opponents that were dealt damage that isnt combat damge. did one damage with dragon tempest and could remove multiple opponents permanents (reported by yaba777 / 2018-08-26 18:20:02.872).
- [ ] Varchild still broken: So varchild makes tokens now but just gives them to yourself instead of your opponent (reported by yaba777 / 2018-08-26 21:10:48.847).
- [ ] Brass Herald: Brass Herald put lands into hand (reported by Biest / 2018-08-26 21:38:50.44).
- [ ] Please fix a card - Contaminated Ground: Instead of my opponent losing 2 live, I've lost 2 lives.Other than that - much love to devs (reported by koshrag / 2018-08-26 22:57:59.376).
- [ ] Yuriko still doesn't work: The commander ninjitsu on Yuriko from Commander 2018 still doesn't work. It asks you to pay the ninjitsu cost of UB and then just untaps the lands you used to pay the cost and does nothing. (reported by JHard / 2018-08-27 00:54:30.019).
- [ ] myth unbound not working: the enchantment does nothing at all... nor discounts spells, nor drawing ability... nothing (reported by capeaux / 2018-08-27 01:02:16.454).
- [ ] Yuriko Still Broken: While yuriko does now have commander ninjitsu, I still can't activate her ability when I have an unblocked attacker, it lets me start to pay for it, then rewinds when I finish. (reported by Kylie / 2018-08-27 01:15:55.206).
- [ ] Xantcha + Propaganda: You can`t skip the attack if all enemies are having Propaganda up (reported by Termaran / 2018-08-27 22:49:27.803).
- [ ] i cant roll back: when i rollback my oponent skip his turn (reported by whiteraven / 2018-08-28 04:12:44.163).
- [ ] lord wingrace is wrong: the text of lord windgrace its worn said zero ladn permantes in last hability (reported by whiteraven / 2018-08-28 04:13:23.935).
- [ ] Gaea's Leige is broken: When it Gaea dies all the lands remain as forests meaning it made him way better than my opponent could account for (reported by 1800SpamMeDad / 2018-08-28 05:52:57.26).
- [ ] cards bugs: Myth unbound does nothing and Nykthos,shrine to nyx does not count transformed permanents for mana with devotion, while it would.is it possible to fix this? thank you (reported by capeaux / 2018-08-28 17:42:34.464).
- [ ] Lili Ult is bugged: With leyline of sanctity out my opponent can target himself and sac MY permanents somehow. Kinda tilting :( (reported by Iburakema / 2018-08-29 09:25:25.805).
- [ ] CRIMSON WISPS: Crimson Wisps from Shadowmoor is a Common and it's coded as an Uncommon. (reported by Bmchafa / 2018-08-29 19:16:53.806).
- [ ] Yuriko isn't working: Trying to activate Yuriko's ninjitsu from the command zone makes the game rollback and give me back the mana without activating the effect. (reported by Metzgeray / 2018-08-30 11:55:22.231).
- [ ] Mind Bomb Does Not Work Correctly: Mind Bomb only does damage to its caster rather than bring symmetrical (reported by Ruby / 2018-08-30 19:50:00.608).
- [ ] Yuriko's ninjitsu: Yuriko's ninjitsu does not work after she dies once on the battlefield. the option to cast is still there, but when you pay for the cost, it just auto cancels and nothing happens. (reported by GaleOfYork / 2018-08-30 20:32:45.761).
- [ ] Thankful for a card: Hello, i would be very thankful if you can add the Card "Gaze of Pain" to the system, so i can use it in EDHTHank (reported by Skymo / 2018-08-31 23:25:25.39).
- [ ] Nullstone Gargoyle counters creatures multiple times: When playing Molimo, Maro Socerer, Nullstone constantlt counters it even though it's a creature and Nullstone is only supposed to affect the first noncreature spell in a turn. (reported by Nonexistant / 2018-09-01 07:29:54.32).
- [ ] Thawing Glacier doesn't return to hand: After using Thawing Glacier's ability, the land doesn't return to the owner's hand. (reported by Nonexistant / 2018-09-01 07:31:08.153).
- [ ] Garruk Relentless does not work as intended: Loyalty counters beeing removed from Garruk Relentless without him being dealt damage (for example via Heart of Kiran) do not cause his first ability to trigger.It probably is not implemented as the state based triggered ability it actually is. (reported by TheAnybody / 2018-09-01 16:18:16.132).
- [ ] Retreat to coralhelm: Can't tap oppenents creatures. (reported by JuakkoL / 2018-09-01 18:03:30.559).
- [ ] Buy more servers: STOP THE SERVERS PROBLEMS,LAGS. is the most important thing in this game. do something.. very frustating very very very (reported by EternalMaster / 2018-09-01 22:38:51.158).
- [ ] Bugs: It is the player damaged by Varchild to get Survivor's tokens, not the Varchild's controller.. (reported by Uomoconodor / 2018-09-02 11:57:40.185).
- [ ] Brass Herald malfunction: I played Brass Herald, and choose Eldrazi as creature type. It revealed four non-Eldrazi cards from top of the deck, but added all four to my hand instead of the graveyard (reported by OzzRamirez / 2018-09-03 03:02:39.183).
- [ ] graphic bug: I was watching a match and switched panel back to see open tables and the targeting arow showed up from teh game on the tables menu (reported by Scapegaot / 2018-09-05 01:31:45.199).
- [ ] oreskos explorer doesn't recognize duals: oreskos explorer is restricted to basic lands this is a bug (reported by StormCroe / 2018-09-05 14:25:50.73).
- [ ] waiting for dead player: a player conceded and priority was later passed to him, locking the game. even after timing out it was locked (reported by Knighthawke / 2018-09-06 06:24:29.163).
- [ ] bug: one of the people we where playing with quit and it said that he was still in the game and beacause of that the rest of us couldnt finish the game and had to concede.also the rollbacks dont work (reported by superjo25 / 2018-09-06 06:26:44.684).
- [ ] Changing Starting Life: I would enjoy to have a feature that can change your starting life, for formats like freeform for example, to play 2vs1 or "King" or some other funny formats. I even thought about pen and paper for xmage :PPlease reply (reported by Fugue91 / 2018-09-06 18:31:13.753).
- [ ] Fatigue: The card "Fatigue" from "Urza's Destiny" is not available. (reported by Badasstodon / 2018-09-06 23:25:06.058).
- [ ] Suggest to Fix: During Sideboarding if you watching a game you being kicked out (reported by MdFkr94 / 2018-09-06 23:54:18.743).
- [ ] Download or Implement Numbers in cards cost: I don't want to download all the images for cards but because of that i don't know the cost of the card can you add if player doesn't have images so that it display numbers or ability to download only mana costs (reported by h0ly_danc3r / 2018-09-07 13:18:16.268).
- [ ] one weith the machine bug: opponent drew 5 cards when his highest cmc was 3 (reported by Knighthawke / 2018-09-07 16:01:13.03).
- [ ] Teferi BUG / Dominaria: Theres an Domenarie Teferi Bug: +1 -> when u have tapped no Lands: The End of Trigger let choose u two, but when you have no tapped -> NO Ok. NO Cancel. (reported by vllprf / 2018-09-08 11:33:04.829).
- [ ] Invalid deck: Says my mono red deck is an invalid color to play commander with. Please advise. (reported by ortingkm / 2018-09-09 04:24:05.66).
- [ ] Varchild, Betrayer of Kjeldor: This card creates the tokens for the attacking player, not the defending playerThe tokens should be given to the player Varchild is attacking. (reported by Boondoggle / 2018-09-09 07:58:45.102).
- [ ] Windgrace's Judgement: Only destroys the first target chosen, not any other targets chosen after. (reported by Boondoggle / 2018-09-09 08:00:00.735).
- [ ] Brain Maggot trouble: I didnt get my exiled card back when my opponents Brain Maggot entered his graveyard from the battlefield.He used it to double block one bestowed creature I attacked with, his two creatures died. (reported by Xanod / 2018-09-09 15:53:51.879).
- [ ] Brass Herald draws all cards: Brass Herald managed to show the top 4 cards of my library in a commander game, only one of these was a Vampire (Ruthless Cullblade), however the other 3 (Vic of Night, Vanq's Banner and Swamp) also got drawn. Possible bug? (reported by Robopig61 / 2018-09-09 21:23:37.149).
- [ ] Option to sort formats Dominaria,Zendicar,... by release date: Option to sort formats Dominaria,Zendicar,... by release date (reported by C0deMaster / 2018-09-10 19:23:05.475).
- [ ] Brudiclad, Telchor Engineer is broken: it clearly says "may chose" but I can not abort this in gameDie Karte sagt kann, aber nicht muss.Thanks a lotDankeschön (reported by AustrianErb / 2018-09-12 00:43:20.598).
- [ ] Add Custodi Lich missing ability: Custodi Lich's "When you become the monarch" ability does not trigger. (reported by Collider / 2018-09-12 04:12:31.463).
- [ ] Dream Leash: my opponent casted a Dream leash on my untapped Liliana of the last hope.the card says"Enchant permanentYou can't choose an untapped permanent as this spell's target as you cast it.You control enchanted permanent."must be a bug (reported by Trilex / 2018-09-12 11:22:19.004).
- [ ] Varchild gives its Controllers the Survivor Tokens: Heya, someone struck another player with Varchild and then Varchild's controller made the Survivor tokens rather than the player who took the damage making them. (reported by Peacemaker250 / 2018-09-13 02:11:12.898).
- [ ] jodha and the cavern of souls: when i play the jhoda hability i can pay the 5 mana colors whith a cavern of souls? (reported by whiteraven / 2018-09-13 20:47:12.41).
- [ ] stonehewer giant and shroud: stonehewer giant ability can attach to a shrouded creature as it doesn't target (reported by StormCroe / 2018-09-14 08:42:44.662).
- [ ] Missing card images: I cant seem to find the promo artwork via images option (reported by Hygaradon / 2018-09-14 23:08:47.759).
- [ ] gods die from toxic deluge: karametra on board, as a creature but didn't die to toxic deluge when x was sufficient to kill it (reported by StormCroe / 2018-09-14 23:09:21.366).
- [ ] Death-Mask Duplicant will crash the game: (300 characters max) (reported by Termaran / 2018-09-15 21:10:32.63).
- [ ] Varchild extremely bugged: "Varchild, Betrayer of Kjeldor" produces the tokens, but for the player who's attacking with it, not for the attacked person.Ein rießen Probem (reported by AustrianErb / 2018-09-17 00:33:58.915).
- [ ] Varchild, Betrayer of kjeldor bug: survivors are supposed to be created under the control of the player who was dealt damage by varchild. Currently it creates the survivors under varchilds controllers control (reported by RessiksLegacy / 2018-09-17 00:40:59.913).
- [ ] Mirrage Mirror blocks the game: I tried to copy oblivion ring with "Mirrage Mirror" during an attack but it blocks the game so that I had to disconnect.Ich habe "Mirrage Mirror" während eines Angriffs vom Gegner mit "Oblivion ring" zu kopieren und das Spiel hat sich aufgehängt. (reported by AustrianErb / 2018-09-17 15:38:35.596).
- [ ] Mistveil Plains: Card dosn't work with it's ability (reported by TonyTab / 2018-09-17 17:53:05.293).
- [ ] Blood Sun: Lands like Cloudpost still come into play tapped with Blood Sun in play. Thats wrong with the new rules. (reported by magicmo / 2018-09-17 18:54:55.506).
- [ ] bug fixing: Hi, i`ve been having problems with lqands that have 2 abilities, like cavern of souls and nykthos. when i click them in a game to activate their second ability, it just adds colorless to my mana pool. I already tried re installing xmage and didint work (reported by Timbauros / 2018-09-17 20:42:09.591).
- [ ] multiplayer game quit bug: are you guys ever gona fix that bug? i.e someone quits without conceding in a multiplayer game and all their permanents remain in effect. because it ruins alot of games. (reported by igniteaxiom / 2018-09-18 06:03:34.389).
- [ ] Declare domination on Bristling Boar: Hello,when cast declare domination on Bristling Boar game is crushed. (reported by Bullet / 2018-09-18 18:30:48.79).
- [ ] Mana Reflection: Mana Reflection when used with somthing like command tower when your commander has multiple colors adds double for each color in the comanders colors. Example- mana refelections tap for green my commander is the mieoplazme it adds 4+ mana instead of 2 (reported by Sexi / 2018-09-19 05:24:34.182).
- [ ] Bug: Beastmasters Magemark didnt work. Attacked with enchanted Petrified wood-kinwas doubleblocked. So supposed to be 6/6 died for 5 dmg.Thx for great program! (reported by _Ezekiel_ / 2018-09-19 14:09:28.156).
- [ ] Varchild BUg: When she attacks she is not giving the opponent any creatures,instead she is giving them to you (reported by Termaran / 2018-09-19 18:38:26.859).
- [ ] Nivmagus Elemental and copies: Nivmagus elemental doesn't let you exile spell-copies from effects like storm (reported by LasseAdelfred / 2018-09-20 03:43:20.202).
- [ ] Celestial Convergence: It removed all omen counters in one turn. (reported by Lazman310 / 2018-09-20 04:11:05.583).
- [ ] swords to plowshares is not exiling creatures but sending them to graveyard: (300 characters max) (reported by Francesco952 / 2018-09-20 14:14:48.598).
- [ ] make it so i have to click less: I have to click 6-7 times for a card to activate, and when they do come out, they're tapped. (reported by TinkerTyler / 2018-09-20 19:00:50.598).
- [ ] player needs removed: 3:42 PM: shinishami: dont play with yanni hes gay (reported by bentspace / 2018-09-20 22:43:55.746).
- [ ] Bug involving Kess and Fire/Ice: I experienced this bug on the xmage.de server. I played Kess, Dissident Mage and then attempted to cast Fire/Ice from my graveyard but it wouldn't let me. I suspect this may be an issue with all split cards and Kess. (reported by JHard / 2018-09-20 22:55:18.666).
- [ ] propaganda: Propaganda doesnt work. Opponent had 2 out, one enchanted by an Estrid Mask token. Neither triggered on an attack (reported by Zox / 2018-09-20 23:36:36.487).
- [ ] card not working: propaganda is not triggering (reported by bentspace / 2018-09-20 23:41:50.24).
- [ ] windgraces judgement: i'm sure you guys already know (reported by igniteaxiom / 2018-09-20 23:42:10.2).
- [ ] Windgrace's Judgement: The card doesn't work properly, it destroys one nonland permanent instead of more (reported by Stahlwolle / 2018-09-21 11:35:29.711).
- [ ] Players leaving without conceding: Players who are leaving without conceding leaves all permanents he or she owns on the battlefield. The main idea here is that those permaments are removed when a player leaves xmage (reported by Stahlwolle / 2018-09-21 11:37:06.121).
- [ ] Server location: Why are there players from brazil and usa when there is a specific server in the usa? (reported by Stahlwolle / 2018-09-21 11:38:17.211).
- [ ] counters on permanents: Sometimes it happens that counters on permanents are unreadable is there a fix? (reported by Stahlwolle / 2018-09-21 11:43:21.335).
- [ ] Bug: The card sylvan reclamation does not work properly. It clearly doenst remove two artifacts and/or enchantments it only removes one kind. fix please (reported by Stahlwolle / 2018-09-21 11:56:38.306).
- [ ] Yuriko: Commander Ninjitsu isn'r working most of the time (reported by FrankDammit / 2018-09-21 17:09:31.164).
- [ ] Copy effects on Glorybringer don't work properly.: When I resolved Phantasmal Image copying Glorybringer, I was able to attack with it and exert it, but the damage ability never triggered. I imagine it's broken for copy effects. (reported by MobiusCipher / 2018-09-22 00:23:36.492).
- [ ] Missing Card-Onslaught Set: Onslaught set is missing Kaboom! http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid=39559 (reported by Kurokikaze / 2018-09-22 07:34:06.466).

**Comment by awjackson** [comment]
> Lili Ult is bugged: With leyline of sanctity out my opponent can target himself and sac MY permanents somehow

Created separate issue #8807 for this since evidently these ancient "feedback checklist" issues never get looked at anymore.
